src/tls_wrap.cc

```diff
@@ -561,7 +561,7 @@ int TLSWrap::DoWrite(WriteWrap* w,
     }
   if (empty) {
     ClearOut();
-    // However if there any data that should be written to socket,
+    // However if there is any data that should be written to socket,
```

Comment:
this is an improvement but there are still two issues in the comment here (ex 'socket' -> 'the socket', 'callback' -> 'the callback'), would you mind fixing the rest while you're here? :)